tf_encrypted/keras/backend.py

```diff
@@ -0,0 +1,34 @@
+"""TFE Keras backend"""
+import tensorflow as tf
+from tensorflow.keras import backend as K
+
+import tf_encrypted as tfe
+from tf_encrypted.protocol.pond import PondPrivateVariable
+
+
+# TODO Should be moved outside backend, maybe in pond.py?
+def is_variable_initialized(x):
+  """Test if TFE private variable has been initialized"""
+  assert isinstance(x, PondPrivateVariable), type(x)
+  is_initialized = tf.is_variable_initialized(x.share0.variable)
```

Comment:
don't think it's a good idea to break encapsulation here. a better approach would be to move this logic to the variables themselves and simply call `x.is_initialized`